lib/global-styles.php

```diff
@@ -635,6 +635,12 @@ function gutenberg_experimental_global_styles_get_editor_features( $config ) {
 		}
 		$features['global']['lineHeight']['custom'] = true;
 	}
+	if ( get_theme_support( 'experimental-custom-spacing' ) ) {
+		if ( ! isset( $features['global']['spacing'] ) ) {
+			$features['global']['spacing'] = array();
+		}
+		$features['global']['spacing']['custom'] = true;
+	}
```

Comment:
This is technically not needed since it's experimental and we're not obliged to ensure backward compatibility here but it's harmless so we can keep both experimental APIs for the moment. We can remove it once theme.json is stable if needed.